### PR TITLE
[Fixes #82] Add image visualizer component

### LINKED ISSUE #82: [Component] base64 image visualizer
https://github.com/intelowlproject/IntelOwl/pull/370 would add an analyzer that sends an image (as a `base64` string) in the analyzer report.

Requirement - a component that would use the `base64` string and display it as an image.

### PR DESCRIPTION
**Pull request by sp35**
This adds a component for the display of an image result  - either base64 or from a URL.

Fixes #82

### CODE REVIEW

File: src/app/pages/dashboard/job-result/job-result.component.ts [hunk 1]
```diff
@@ -220,6 +223,9 @@ export class JobResultComponent implements OnInit, OnDestroy {
       ? event.data.report
       : event.data.errors;
     this.editor.update(json);
+
+    if ('screenshot' in json && json['screenshot'].length)
```
Comment (Eshaan7) [inline src/app/pages/dashboard/job-result/job-result.component.ts:227]: using the `Object.prototype.hasOwnProperty` to check for key would be a more "javascript" approach, I believe.

**Comment by Eshaan7** [comment]
Very good work!

**Comment by Eshaan7** [comment]
Damn, I just noticed that you had made this PR against `master` but it was to be made against `develop`. My fault for not noticing this before merging it :/

Anyway, I just merged `master` with `develop` now to fix this.

**Comment by sp35** [comment]
Sorry, my bad!

**Pull request merged**